backend/src/common/types/where-to-buy/where-to-buy.params.ts

```diff
@@ -0,0 +1,18 @@
+interface AutoriaResponse {
+  additional_params: { page: string };
+  result: {
+    search_result: {
+      ids: string[];
+      count: number;
+      last_id: number;
+    };
+  };
+}
+
+type WhereBuyRequestQuery = {
+  page: string;
+  id: string;
+  countpage: string;
```

Comment:
It's a good practice to put props in the types from more-> less specific order. In this case, `id` is the most specific field possible and should be first